src/components/GetClimbForm.js

```diff
@@ -45,13 +46,13 @@ function GetClimbForm(props) {
     return <>
         {props.title && <div>Edit a Climb</div>}
         {!climb && <>
-            <input placeholder='Rope' value={rope} onChange={(e) => setRope(e.target.value)} />
-            <input placeholder='Colour' value={colour} onChange={(e) => setColour(e.target.value)} onBlur={() => handleClimbChanged()} />
+            <TextField id="outlined-basic" label="Rope" variant="outlined" placeholder='Rope' value={rope} onChange={(e) => setRope(e.target.value)} />
```

Comment:
Duplicated IDs. Not good. Consider classes instead.

Reply:
Removed ID